data/hai/hai reveal 4 middle.txt

```diff
@@ -178,13 +178,11 @@ mission "Hai Reveal [A03a] Symbol Identification"
 		has "Hai Reveal [A02] Symbol Identification: done"
 	on offer
 		event "alondo symbol timer" 12
-		log `Been asked to check out the Hai spaceports in various systems near the wormhole, just to check nothing unexpected is happening.`
+		log `Alondo asked me to check out the Hai spaceports in various systems near the wormhole, to see if anything unexpected is happening.`
 		conversation
-			`You make what is beginning to feel like a standard request to land at the secure spaceport and disembark with relevant copies of files on a data stick in hand. It doesn't take long to track down Alondo, though you do end up having to wait for nearly half an hour while he finishes up a meeting with a Hai official of some kind. When he's finished he waves you into the room and gestures for you to take a seat. The seat you choose is clearly designed to be comfortable for a Hai, making for a rather strange experience for a human.`
-			# TODO: Replace this paragraph with actual content. Include something about him working at Remington lately.
-			`	You show Alondo the cat symbol you found, and he says, "I know some researchers who can be discreet. I'll ask them to look into this."`
-			`	"In the meantime," he says, "could I ask you to stop by some of the Hai worlds nearer the wormhole? They have obviously ramped up security on Allhome, but I'd like to have a pair of eyes check that they're not using that as a cover to move more military through other nearby worlds into a threatening position... not that I think they'd do that, but we've just been through one war. I think it's making me a little paranoid while I sit here in the most Hai place that there is."`
-			`	He shrugs. "Plus, they say they're ok, but it would still make me feel more secure if someone else on our side would check on them. It will probably be a couple of weeks till I can track someone down for you. So, go check on that for me, and I'll let you know."`
+			`You make what is beginning to feel like a standard request to land at the secure spaceport and disembark with relevant copies of files on a data stick in hand. It doesn't take long to track down Alondo, though you do have to wait nearly half an hour while he finishes a meeting with a Hai official. When he's finished, he waves you into the room and you take a seat. The seat you choose is clearly designed to be comfortable for a Hai, making a strange experience for a human.`
```

Comment:
```suggestion
			`You make what is beginning to feel like a standard request to land at the secure spaceport and disembark with relevant copies of files on a data stick in hand. It doesn't take long to track down Alondo, though you do have to wait nearly half an hour while he finishes a meeting with a Hai official. When he's done, he waves you into the room and asks you to take a seat. The seat you choose is clearly designed to be comfortable for a Hai, making for a rather strange experience for a human.`
```

I put a few things back. Removing that little "of some kind" made the repetition more visible.